### ISSUE TITLE
Traefik with Run-AlValidation

### ISSUE THREAD

**Issue by catadumitru** [microsoft/navcontainerhelper#1517]
**Describe the issue**
I'm trying to run the Run-AlValidation command on a server where I have traefik installed but get an error,
"Using Traefik only makes sense if you allow external access, so you have to provide the public DNS name (param -PublicDnsName)"

Is there a way to set the PublicDnsName when running this command or have the -useTraefik:$false by default?
**Scripts used to create container and cause the issue**
```
$validationResult = Run-AlValidation -licenseFile $licenseFile `
                                     -previousApps $previousApps `
                                     -apps $apps `
                                     -validateCurrent `
                                     -countries $countries `
                                     -affixes $affixes `
                                     -supportedCountries $countries `

**Output of scripts**
```
Creating container for country us
BcContainerHelper is version 1.0.15
BcContainerHelper is running as administrator
Host is Microsoft Windows Server 2019 Datacenter - ltsc2019
Docker Client Version is 19.03.5
Docker Server Version is 19.03.5
Downloading application artifact /sandbox/17.1.18256.19737/us
Downloading C:\Users\CD\AppData\Local\Temp\7\4cf8023b-8f26-4f9d-91b9-373d90e4c3ff.zip
Unpacking application artifact to tmp folder using Expand-Archive
Downloading platform artifact /sandbox/17.1.18256.19737/platform
Downloading C:\Users\CD\AppData\Local\Temp\7\98a89e18-927a-468b-8403-ae8830ef1784.zip
Unpacking platform artifact to tmp folder using Expand-Archive
Downloading Prerequisite Components
Downloading c:\bcartifacts.cache\sandbox\17.1.18256.19737\platform\Prerequisite Components\Open XML SDK 2.5 for Microsoft Office\OpenXMLSDKv25.msi
Downloading c:\bcartifacts.cache\sandbox\17.1.18256.19737\platform\Prerequisite Components\IIS URL Rewrite Module\rewrite_2.0_rtw_x64.msi
Downloading c:\bcartifacts.cache\sandbox\17.1.18256.19737\platform\Prerequisite Components\DotNetCore\DotNetCore.1.0.4_1.1.1-WindowsHosting.exe
Fetching all docker images
WARNING: useTraefik not specified, but Traefik container was initialized, using Traefik. Specify -useTraefik:$false if you do NOT want to use Traefik.
Enabling SSL as otherwise all clients will see mixed HTTP / HTTPS request, which will cause problems e.g. on the mobile and modern windows clients
                                      
Removing container took 1 seconds
Using Traefik only makes sense if you allow external access, so you have to provide the public DNS name (param -PublicDnsName)
At C:\Program Files\WindowsPowerShell\Modules\BcContainerHelper\1.0.15\ContainerHandling\New-NavContainer.ps1:589 char:13
+             throw "Using Traefik only makes sense if you allow extern ...
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Using Traefik o...-PublicDnsName):String) [], RuntimeException
    + FullyQualifiedErrorId : Using Traefik only makes sense if you allow external access, so you have to provide the public DNS name (param -PublicDnsName)
```

**Comment by freddydk** [comment]
Looked at this.
The reason for this problem (as you know) is that the publicDnsName of the computer is not specified.

You can create a text file called C:\inetpub\wwwroot\hostname.txt with you public dns name inside, then that will be picked up by New-BcContainer and used as PublicDnsName and containers will automatically get traefik.

Or you can get the next version of containerhelper where I have modified New-BcContainer to ONLY default to traefik if publicDnsName is specified or the hostname.txt file is present, meaning in Run-AlValidation it will not add traefik to the containers and it should also not be needed.

**Comment by freddydk** [comment]
Fix shipped in 1.0.16
